application/language/spanish/qso_lang.php

```diff
@@ -26,74 +26,74 @@
 $lang['qso_dok_helptext'] = 'Por ejemplo: Q03';
 
 $lang['qso_notes_helptext'] = 'El contenido es usado solo dentro de Cloudlog y no es exportado a otros servicios.';
-$lang['qsl_notes_helptext'] = 'This note content is exported to QSL services like eqsl.cc.';
+$lang['qsl_notes_helptext'] = 'El contenido de esta nota es exportado a servicios QSL como eqsl.cc.';
 
-$lang['qso_eqsl_qslmsg_helptext'] = "Get the default message for eQSL, for this station.";
+$lang['qso_eqsl_qslmsg_helptext'] = "Obtener el mensaje por defecto para eQSL, para esta estación.";
 
 // error text //
-$lang['qso_error_timeoff_less_timeon'] = "TimeOff is less than TimeOn";
+$lang['qso_error_timeoff_less_timeon'] = "TimeOff es menor que TimeOn";
 
 // Button Text on /qso Display
 
-$lang['qso_btn_reset_qso'] = 'Resetear';
+$lang['qso_btn_reset_qso'] = 'Reinicializar';
 $lang['qso_btn_save_qso'] = 'Guardar QSO';
 $lang['qso_btn_edit_qso'] = 'Editar QSO';
-$lang['qso_delete_warning'] = "Warning! Are you sure you want delete QSO with ";
+$lang['qso_delete_warning'] = "¡Advertencia! ¿Está seguro que desea eliminar QSOs con ";
 
 // QSO Details
 
 $lang['qso_details'] = 'Detalles de QSO';
 
-$lang['fav_add'] = 'Add Band/Mode to Favs';
-$lang['qso_operator_callsign'] = 'Operator Callsign';
+$lang['fav_add'] = 'Añadir Banda/Modo a Favoritos';
+$lang['qso_operator_callsign'] = 'Indicativo de Operador';
 
 // Simple FLE (FastLogEntry)
 
-$lang['qso_simplefle_info'] = "What is that?";
+$lang['qso_simplefle_info'] = "¿Qué es esto?";
 $lang['qso_simplefle_info_ln1'] = "Simple Fast Log Entry (FLE)";
-$lang['qso_simplefle_info_ln2'] = "'Fast Log Entry', or simply 'FLE' is a system to log QSOs very quickly and efficiently. Due to its syntax, only a minimum of input is required to log many QSOs with as little effort as possible.";
-$lang['qso_simplefle_info_ln3'] = "FLE was originally written by DF3CB. He offers a program for Windows on his website. Simple FLE was written by OK2CQR based on DF3CB's FLE and provides a web interface to log QSOs.";
-$lang['qso_simplefle_info_ln4'] = "A common use-case is if you have to import your paperlogs from a outdoor session and now SimpleFLE is also available in Cloudlog. Information about the syntax and how FLE works can be found <a href='https://df3cb.com/fle/documentation/' target='_blank'>here</a>.";
-$lang['qso_simplefle_qso_data'] = "QSO Data";
-$lang['qso_simplefle_qso_date_hint'] = "If you don't choose a date, today's date will be used.";
-$lang['qso_simplefle_qso_list'] = "QSO List";
+$lang['qso_simplefle_info_ln2'] = "'Fast Log Entry', o simplemente 'FLE' es un sistema para registrar QSO muy rápida y eficientemente. Debido a su sintaxis, solo se requiere un mínimo de campos se requiere para guardar muchos QSOs con la menor cantidad de esfuerzo posible.";
```

Comment:
There is "se require" twice. Consider "solo se requieren pocos campos para introducir muchos QSOs" or similar.